src/components/HonorAward.tsx

```diff
@@ -65,7 +66,7 @@ const HonorAwards = () => {
     // Get data from data base
     //***********************************************************/
     const getData = async () => {
-        axios.get("http://3.236.213.150:8081/honor/portfolio/"+cookies['portfolio'].id)
+        axios.get(url + "/honor/portfolio/"+cookies['portfolio'].id)
```

Comment:
@ddinh please store url + "/honor" to a variable and pass here to minimize code duplication, throughout this file